src/Notifications/NotificationCenterController.cs

```diff
@@ -72,17 +79,56 @@ internal NotificationCenterController(DynamoView view, DynamoLogger dynLogger)
                 HorizontalOffset = notificationPopupHorizontalOffset,
                 VerticalOffset = notificationPopupVerticalOffset
             };
-            notificationUIPopup.webView.EnsureCoreWebView2Async();
-            notificationUIPopup.webView.CoreWebView2InitializationCompleted += WebView_CoreWebView2InitializationCompleted;
             logger = dynLogger;
             // If user turns on the feature, they will need to restart Dynamo to see the count
             // This ensures no network traffic when Notification center feature is turned off
             if (dynamoViewModel.PreferenceSettings.EnableNotificationCenter) 
             {
                 RequestNotifications();
+                InitializeBrowserAsync();
+            }
+        }
+
+        private void View_Closing(object sender, System.ComponentModel.CancelEventArgs e)
+        {
+            dynamoView.Closing -= View_Closing;
+            SuspendCoreWebviewAsync();
+        }
+
+        async void SuspendCoreWebviewAsync()
+        {
+            notificationUIPopup.IsOpen = false;
+            notificationUIPopup.webView.Visibility = Visibility.Hidden;
+
+            if (notificationUIPopup.webView.CoreWebView2 != null)
+            {
+                await notificationUIPopup.webView.CoreWebView2.TrySuspendAsync();
```

Comment:
why even do this, it appears to sleep the webview, when we call stop on the next line?